testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -184,11 +185,40 @@ def test_charge_parse(self, universe):
 
     def test_mass_parse_or_guess(self, universe):
         assert_almost_equal(universe.atoms[0].mass, 8.0)
-        assert_almost_equal(universe.atoms[1].mass, 20.98)
-        assert_almost_equal(universe.atoms[2].mass, 20.98)
+        assert_almost_equal(universe.atoms[1].mass, 20.989)
+        assert_almost_equal(universe.atoms[2].mass, 20.989)
         assert_almost_equal(universe.atoms[3].mass, 1.008)
 
 
+class TestITPAtomtypes(ParserBase):
+    parser = mda.topology.ITPParser.ITPParser
+    ref_filename = ITP_charges
+    expected_attrs = ['ids', 'names', 'types', 'masses',
+                      'charges', 'chargegroups',
+                      'resids', 'resnames',
+                      'segids', 'moltypes', 'molnums',
+                      'bonds', 'angles', 'dihedrals', 'impropers']
+    guessed_attrs = []
+    expected_n_atoms = 9
+    expected_n_residues = 3
+    expected_n_segments = 1
+
+    @pytest.fixture
+    def universe(self, filename):
+        return mda.Universe(filename)
+
+    def test_charge_parse(self, universe):
+        assert_almost_equal(universe.atoms[0].charge, -1.0)
+        assert_almost_equal(universe.atoms[1].charge, 0)
+        assert_almost_equal(universe.atoms[2].charge, 0)
+        assert_almost_equal(universe.atoms[3].charge, -1.)
+
+    def test_mass_parse_or_guess(self, universe):
+        assert_almost_equal(universe.atoms[0].mass, 100.0)
+        assert_almost_equal(universe.atoms[1].mass, 100.0)
+        assert_almost_equal(universe.atoms[2].mass, 100.0)
+        assert_almost_equal(universe.atoms[3].mass, 100.0)
```

Comment:
since it is used a lot here, perhaps worth noting that `assert_allclose` is recommended these days per the docs:
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html

Reply:
@tylerjereddy - do you think that warrants a proper issue being raised? i.e. is there a possibility that assert_almost_equal will eventually be deprecated?

I'll raise a userguide issue since we recommend assert_almost_equal at the moment.

Reply:
Ok, I'll update the tests I've added and also convert the other ones in the test_itp to use `assert_allclose`. I was just following the existing convention in the file but happy to change it.